packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 2.4.0 - Aug 9 2021
+
+Incremental version for Teams Toolkit with multiple bugs fixed and the following updates：
+
+New Feature:
+- Add extension accessiblity for the disabled.
+- Add CLI command 'teamsfx preview' to preview SPFx apps.
+
+Enhancement:
+- A/B testing for Tree View design. You may randomly see either one of the two Tree View layouts in the Side bar. See below picture.
+  
+  ![TreeViewABtest](./media/TreeViewABtest.png)
```

Comment:
Seems the image doesn't exist.

Reply:
@qinezh  it's good from my side. Can you double confirm this from your side?

Reply:
@IvanJobs yeah, looks the functionality to display rich diff is not accurate... Resolved.

Reply:
@IvanJobs Is local resource supported here? I remember when I reference image in README.md only link in following format is supported. 
https://raw.githubusercontent.com/OfficeDev/TeamsFx/main/packages/vscode-extension/media/xx.png